rasa/nlu/training_data/training_data.py

```diff
@@ -476,10 +476,18 @@ def _split(_examples: List[Message], _count: int) -> None:
         return test, train
 
     def print_stats(self) -> None:
+        number_of_examples_for_each_intent = []
+        for key, value in self.number_of_examples_per_intent.items():
```

Comment:
A tiny bit more explicit:
```suggestion
        for intent_name, example_count in self.number_of_examples_per_intent.items():
```